example/app/config.yml

```diff
@@ -50,6 +50,8 @@ default:
   activities:
     - name: EcsTaskHelloWorld
 
+  system_bucket: cumulus-test-sandbox-internal
+
   buckets:
     internal: cumulus-test-sandbox-internal
```

Comment:
This is not important, just a recommendation:

to avoid repetition you can change `internal: cumulus-test-sandbox-internal` to `internal: '{{system_bucket}}'`